tests/cut-plane/test_detection.py

```diff
@@ -155,7 +155,7 @@ def _assert_expected_plane(plane):
 
 def test_find():
     filename = DATA / 'rotated.h5'
-    neuron = nm.load_neuron(filename)
+    neuron = nm.io.utils.load_morphology(filename)
```

Comment:
nm.load_morphology() works as well

Reply:
ok, I'll switch them; the deprecation message mentioned this